src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    private static final String MESSAGE_PASSWORD_HAS_NULL = "password or repeatPassword is null";
+    private static final String MESSAGE_PASSWORDS_ARE_WRONG = "Wrong passwords";
+
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null) {
+            throw new PasswordValidationException(MESSAGE_PASSWORD_HAS_NULL);
+        }
```

Comment:
This is redundant. As Kseniia mentioned, `repeatPassword == null` is managed in `password.equals(repeatPassword)` when `password == null` is checked before it. So, you can delete this check and add `password == null` at the start of the next one